packages/unified-selection/README.md

```diff
@@ -166,3 +166,21 @@ const queryExecutor = createECSqlQueryExecutor(imodel);
 // Returns the parent element, or the element itself if it does not have a parent, for each element specified in `elementIds` argument.
 const selection = computeSelection({ queryExecutor, elementIds, scope: { id: "element", ancestorLevel: 1 } });
 ```
+
+## iModel selection synchronization with unified selection
+
+The `@itwin/unified-selection` package delivers a `enableUnifiedSelectionSyncWithIModel` function to enable selection synchronization between an iModel and a `SelectionStorage`. When called it returns a cleanup function that should be used to disable the synchronization. There should only be one active synchronization between a single iModel and a `SelectionStorage` at a given time. For example, this function could be used inside a `useEffect` hook in a viewport component but should not be used if there is more than one viewport for the same iModel (in such cases the function could be called in a higher scope so that all viewport components would share the synchronization):
```

Comment:
```suggestion
The `@itwin/unified-selection` package delivers a `enableUnifiedSelectionSyncWithIModel` function to enable selection synchronization between an iModel and a `SelectionStorage`. When called, it returns a cleanup function that should be used to disable the synchronization. There should only be one active synchronization between a single iModel and a `SelectionStorage` at a given time. For example, this function could be used inside a `useEffect` hook in a viewport component but should not be used if there is more than one viewport for the same iModel (in such cases the function could be called in a higher scope so that all viewport components would share the synchronization):
```

Reply:
Fixed.